api/src/test/java/org/pmiops/workbench/interceptors/AuthInterceptorTest.java

```diff
@@ -47,95 +59,114 @@ public void handle() {}
   public void handle() {}
 }
 
+@RunWith(SpringRunner.class)
 public class AuthInterceptorTest {
 
   private static final long USER_ID = 123L;
 
-  @Mock private UserInfoService userInfoService;
-  @Mock private FireCloudService fireCloudService;
-  @Mock private UserDao userDao;
+  @MockBean private UserInfoService userInfoService;
+  @MockBean private FireCloudService fireCloudService;
+  @MockBean private UserDao userDao;
+  @MockBean private UserService userService;
+  @MockBean private DevUserRegistrationService devUserRegistrationService;
+
+  private static WorkbenchConfig workbenchConfig;
+
   @Mock private HttpServletRequest request;
```

Comment:
nit: new convention is to have a `mock` prefix for mock instance variables, since the call site can be a long way from the declaration.

Reply:
👍  I think creating an API style guide will be a helpful place to document conventions like this, especially when they're a change from much of our existing code.

Reply:
I could spend time on that. In most  instances folks are ok making little improvements like that without a guide. I've just been changing them all whenever I edit a test class.